CMakeLists.txt

```diff
@@ -15,6 +15,11 @@ set(ONNX_ROOT ${PROJECT_SOURCE_DIR})
 set(CMAKE_MODULE_PATH "")
 list(APPEND CMAKE_MODULE_PATH ${ONNX_ROOT}/cmake/Modules)
 
+if(ENABLE_GTESTS)
```

Comment:
Add "ONNX" to the flag. Otherwise if onnx's cmake files get included by other frameworks's cmake build system, the parent scope variables will leak to onnx's.